.github/workflows/release.yml

```diff
@@ -183,6 +183,8 @@ jobs:
     steps:
       - uses: actions/checkout@v1
       - uses: bahmutov/npm-install@v1.1.0
+      - name: Install wasm-pack
+        run: curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh
```

Comment:
Could also use the node module https://yarnpkg.com/package/wasm-pack

Reply:
I tried that, and it didn't work (failed with `Not installed: "wasm-pack"` at runtime)

Reply:
https://github.com/parcel-bundler/source-map/runs/2659683400?check_suite_focus=true

Reply:
Oh that's strange, as long as it works it's probably fine